src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
```

Comment:
let's write enum values in uppercase
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
You need to know, that firstly I wrote enum values in UpperCase, but later I rewrote it. I did't know why(